Xamarin.Essentials/Calendar/Calendar.shared.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Calendar
+    {
+        static TimeSpan defaultStartTimeFromNow = TimeSpan.Zero;
+
+        static TimeSpan defaultEndTimeFromStartTime = TimeSpan.FromDays(14);
+
+        public static bool IsSupported => PlatformIsSupported;
```

Comment:
I think we need to remove this based on the feedback on github

Reply:
Removed